### PR TITLE
Updated modelParams and opf code cleanup

### PR DESCRIPTION
**Pull request by breznak**
model params mirror the (best) settings from nupic/examples/anomaly/hotgym. 
Merged and cleaned up the support code in `opf/`

Fixes: #14

### CODE REVIEW

File: opf/anomalyTester.py [hunk 1]
```diff
@@ -1,97 +0,0 @@
-#!/usr/bin/env python
-
-from nupic.frameworks.opf.modelfactory import ModelFactory
```
Comment (breznak) [inline opf/anomalyTester.py:3]: this is complicated :) I've first merged functionality of anomaly_benchmark into anomalyTester, removed the benchmark, and then renamed anomalyTEster to anomaly_benchmark (to increase the confusion). So please see the file below.

File: opf/anomaly_benchmark.py [hunk 2]
```diff
@@ -1,12 +1,128 @@
...
+        self.model.enableInference({'predictedField': 'function'})
+
+        if plotting:
+            raise NotImplementedError('Not yet implemented')
+# plt.ion()
+# fig = plt.figure()
+#
+# plt.title('Testing the model')
+# plt.xlabel('time [s]')
+# plt.ylabel('Y axe')
```
Comment (breznak) [inline opf/anomaly_benchmark.py:57]: removed the `model.py` as it was duplicit to anomalyTester, just copied the plot functionality comments, do we want it? (it does not seem to be working)

File: opf/modelParams.py [hunk 3]
```diff
@@ -65,19 +72,15 @@
...
             'synPermConnected': 0.1,
-
-            'synPermActiveInc': 0.0001,
-
-            # Changed: 'synPermInactiveDec': 0.0005,
-
-            # CHANGED :'maxBoost': 1.0,
+            'synPermActiveInc': 0.1,
+            'synPermActiveInc': 0.01,
+            'maxBoost': 10.0, # 10.0 is default value
```
Comment (breznak) [inline opf/modelParams.py:83]: these params are changed, so we need to rerun the models.

**Comment by breznak** [comment]
@wattik please review

File: opf/anomaly_benchmark.py [hunk 4]
```diff
@@ -1,12 +1,128 @@
...
+            self._write_row(row)
+
+            k = i % 1000
+            if k == 0:
+                print(str(i) + " iterations.")
+
+
+if __name__ == "__main__":
+    import modelParamsRDSE as mpRDSE
+    import modelParams as mp
```
Comment (breznak) [inline opf/anomaly_benchmark.py:122]: the former anomaly_benchmark is now here in main()

File: opf/modelParamsRDSE.py [hunk 5]
```diff
@@ -1,28 +1,10 @@
...
-            # if > 0: sensor region will print out on screen what it's sensing
-            # at each step 0: silent; >=1: some info; >=2: more info;
-            # >=3: even more info (see compute() in py/regions/RecordSensor.py)
-            'verbosity' : 0,
-
-            
-            'encoders': {
+# Override with the changes:
+MODEL_PARAMS['modelParams']['sensorParams']['encoders'] = {
                 u'function':    {
```
Comment (breznak) [inline opf/modelParamsRDSE.py:8]: deduplicating the code here, update just the changes.

File: opf/anomaly_benchmark.py (hunk 2 above)
Comment (wattik) [inline opf/anomaly_benchmark.py:57]: Yes, it was a leftover.

File: opf/modelParamsRDSE.py (hunk 5 above)
Comment (wattik) [inline opf/modelParamsRDSE.py:8]: nice! :+1:

File: opf/anomaly_benchmark.py (hunk 2 above)
Comment (breznak) [inline opf/anomaly_benchmark.py:57]: ok :+1: 
Should we remove the pyplot stuff, or is it working?

**Pull request merged**